test/cmocka/src/common_mocks.c

```diff
@@ -24,3 +65,13 @@ int memcpy_s(void *dest, size_t dest_size,
 
 	return 0;
 }
+
+void WEAK __panic(uint32_t p, char *filename, uint32_t linenum)
+{
+	(void)p;
+	(void)filename;
+	(void)linenum;
+
+	print_error("panic: %s:%d (code %d)", filename, linenum, p);
```

Comment:
print_error can be defined out? If not, the casts to void above aren't needed.

Reply:
yes, casting is redundant.